Sail/src/API/DX12/dxr/DXRUtils.cpp

```diff
@@ -149,7 +149,9 @@ ID3D12StateObject* DXRUtils::PSOBuilder::build(ID3D12Device5* device) {
 
 		hr = device->GetDeviceRemovedReason();
 		_com_error err(hr);
-		std::cout << err2.ErrorMessage() << std::endl;
+		OutputDebugString(err.ErrorMessage());
+		OutputDebugString(L"\n");
+		std::cout << err.ErrorMessage() << std::endl;
```

Comment:
Why not use the Logger?

Reply:
This is ran on dx device removal and crashes application, console is not always available at that point